localstack/services/awslambda/api_utils.py

```diff
@@ -66,6 +77,27 @@
 # Date format as returned by the lambda service
 LAMBDA_DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%f+0000"
 
+# TODO: add nodejs18_x
+RUNTIMES = [
```

Comment:
Can't we use lambda_models.IMAGE_MAPPING keys for this? Seems redundant to have them both defined.

Reply:
to avoid the circular import issue here, we'll tackle this at a later time :+1: